pod/perlgov.pod

```diff
@@ -395,8 +395,8 @@ entire Steering Council is ever vacant, a Term Election will be held instead.
 
 The Steering Council may defer the Special Election for up to twelve weeks.
 Their intent to do so must be publicly stated to the Core Team.  If any active
-Core Team member objects, the Special Election must be organized within one
-week.
+Core Team member objects within one week, the Special Election must be
+organized within one week.
```

Comment:
Given that without deferment the vote must be organized within two weeks, I would have expected an objection to deferment to cause it to fall back to two weeks rather than a tighter one-week schedule.

Reply:
This is fine with me and I have no feelings about it either way.